seqr/management/commands/detect_inactive_privileged_users.py

```diff
@@ -0,0 +1,59 @@
+from anymail.exceptions import AnymailError
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.db.models import Q
+import logging
+
+logger = logging.getLogger(__name__)
+
+WARNING_TEMPLATE = """
+Hi there {} - 
+
+You have not logged in to seqr in {} days. Unless you log in within the next {} days, your account will be deactivated.
+"""
+
+DEACTIVATE_TEMPLATE = """
+Hi there {} - 
+
+You have not logged in to seqr in {} days, and therefore your account has been deactivated.
```

Comment:
Don't you think using named placeholders is better?

Reply:
not really, only when theres a lot of variables or the usage is confusing. In simple cases I think named placeholders just add unnecessary words.  If you think its confusing without named placeholders here I can change it, but I think its fine as is